tests/tiup-cluster/script/tikv_cdc.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+set -eu
+
+function tikv_cdc_test() {
+	mkdir -p ~/.tiup/bin/
+
+	version="nightly"
+	topo_name="tikv_cdc"
+	test_tls=false
+	mirror="https://tiup-mirrors.pingcap.com/"
```

Comment:
`tiup mirror set --reset` will do the same as `tiup mirror set https://tiup-mirrors.pingcap.com`